### DISCUSSION TITLE
Pass data to STDIN?

### DISCUSSION THREAD

**Discussion by codeit-ninja** [deployphp/deployer#3624] [Help needed]
How to pass data to the STDIN using the `run` function? Is that possible? The underlying library (symfony process) supports it.

**Comment by antonmedv** [comment]
Currently, this is not possible. Deployer utilizes STDIN for it's own needs. 

What is your use case?

**Reply by codeit-ninja** [reply]
I was trying to send a mysql dump from STDOUT to STDIN, so I didnt have to implement a solution to create a temp file, upload it to remote and then delete it again from local and remote.

**Reply by antonmedv** [reply]
In this case temp file is needed.
